requirements.txt

```diff
@@ -1,10 +1,11 @@
 hydra-core>=1.2.0
 wandb>=0.13
-tree-sitter
-pygraphviz
-graphviz
 numpy
 transformers>=4.22.0
 tokenizers
+torch
```

Comment:
AFAIK torch shouldn't be needed here since it's a dependency of transformers.

Reply:
I remember I ran into the problem of missing torch when doing fresh install.. Let me double check.

Reply:
@herbiebradley Oh I remember the problem now! In CI it reports the error
`ModuleNotFoundError: No module named 'torch'`
in pytest...